src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -775,5 +775,10 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "POST" }
     ),
+    verifiedRepresentativesLoader: gravityLoader<any, { artist_id: string }>(
+      ({ artist_id }) => `verified_representatives?artist_id=${artist_id}`,
```

Comment:
**question (non-blocking):** The fact that this is defined in an authenticated loader means that this will fail for unauthenticated requests to Gravity. I see that this is necessary now because of the [`admin true`](https://github.com/artsy/gravity/blob/112ffe7907729535a320e14fdc1fe43f039b628f/app/api/v1/verified_representatives_endpoint.rb#L7) requirement at the endpoint; are we planning to change that?

Reply:
Good calls! 🚀